atest/robot/output/xunit.robot

```diff
@@ -89,3 +113,18 @@ Suite Stats Should Be
     Element Attribute Should Be       ${elem}    skipped     ${skipped}
     Element Attribute Should Match    ${elem}    time        ?.???
     Element Attribute Should Be       ${elem}    errors      0
+    Element Attribute Should Match    ${elem}    timestamp   ????-??-??T??:??:??.???000
+
+Verify XUnit Timestamp
+    [Arguments]    ${pattern}
+    File Should Exist    ${OUTDIR}/xunit.xml
+    ${suite} =    Get XUnit Node
+    Should match    ${suite.attrib['timestamp']}    ${pattern}
```

Comment:
This could probably be `Should Be Equal` instead of `Should Match`. For consistence reasons and also otherwise it would be better to use `Element Attribute Should Be` like in the keyword above.